azure-spring-cloud-maven-plugin/src/test/resources/pom-1.xml

```diff
@@ -26,7 +26,7 @@
 		<plugins>
 			<plugin>
 				<groupId>com.microsoft.azure</groupId>
-				<artifactId>azure-spring-maven-plugin</artifactId>
+				<artifactId>azure-spring-cloud-maven-plugin</artifactId>
 				<version>0.0.1-SNAPSHOT</version>
```

Comment:
Shall we add a parent pom or something else to get the correspond spring maven plugin version? Or we need to update every file when bump version

Reply:
It is not good to use variable for artifactId